demo/async-read-file.js

```diff
@@ -1,18 +1,30 @@
 // Run this with
 //
-//   traceur --experimental async-read-file.js file-to-read
+//   traceur --async-functions=true async-read-file.js
 
-function lineCount(file) {
-  var data;
-  try {
-    // TODO: make await an expression.
-    await data = require('fs').readFile(file, 'utf8');
-    console.log(`${file} has ${data.split('\n').length} lines`);
-  } catch (ex) {
-    console.error(ex);
-  }
+function asyncReadFile(file) {
+  return new Promise((resolve) => {
+    require('fs').readFile(file, 'utf8', (err, data) => {
+      if (err)
+        throw err;
+      resolve(data);
+    });
+  });
 }
 
-var fileName = process.argv[2];
-lineCount(fileName);
-console.log(`Loading ${fileName}...`);
+(async function() {
+  // Set filename to demo/async-read-file.js (this file).
+  let filename = __filename;
```

Comment:
Actually I've updated the comment.

When I run:

```
./traceur --experimental demo/async-read-file.js some-file
```

I got:

```
Error: Error: File not found '/Users/rogerio/projects/traceur-compiler/some-file.js'
```

It is trying to compile ```some-file.js```.

BTW, process.argv in this case would be:

```
[ 'node',
  '/Users/rogerio/projects/traceur-compiler/traceur',
  '--experimental',
  'demo/async-read-file.js',
  '/Users/rogerio/projects/traceur-compiler/some-file.js' ]
```

So the right key should be argv[4], but it will still try to compile ```some-file.js```.

Is it expected?